molecule/docker/create.yml

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
Why this file is needed? Why not just link in `molecule.yml` to one from `defaults/` ?

Reply:
regarding the molecule scenario, I basically copied the folder and tried to match the scenario to make molecule build prometheus with docker. I've then adapted things where needed but I've not cleaned anything nor did I had a deep enough understanding on how molecule was configured there. I was basicly modifying the scenario until I hit my dead end with it